package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft.azure/autorest.testserver",
-  "version": "3.2.3",
+  "version": "3.3.0",
```

Comment:
Need to update that one again. Just increase by a patch value when tweaking the tests.